packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -774,10 +790,23 @@ export const Table = <
 
       dispatch({ type: tableResizeStartAction });
     },
-    [dispatch, state.columnResizing.columnWidths, flatHeaders, instance],
+    [
+      dispatch,
+      state.columnResizing.columnWidths,
+      flatHeaders,
+      instance,
+      isResizable,
+    ],
   );
   const [resizeRef] = useResizeObserver(onTableResize);
 
+  const [headerScrollWidth, setHeaderScrollWidth] = React.useState(
+    headerRef.current?.scrollWidth ?? 0,
+  );
+  const [headerClientWidth, setHeaderClientWidth] = React.useState(
+    headerRef.current?.clientWidth ?? 0,
+  );
```

Comment:
These should be initialized to 0. The refs will always be empty on first render, so the code never runs

Reply:
Initialized to 0 (https://github.com/iTwin/iTwinUI/pull/1809/commits/c3c1c1a282e058842a98f227fccee9dffabf6c18).